### PR TITLE
Updated whoogle + jackett + other stuff

### PR DESCRIPTION
**Pull request by bruvv**
- Whoogle is now better configurable.
- Jackett fixed errors in the new config folder (https://github.com/alexbelgium/hassio-addons/issues/135) and streamlined the version
- Made some cosmetic changes to the files
- Added default github files, hopefully changing the EOL for good
- Added gitignore
- Fixed mulitple SC2163 and SC2068
- imgbot did his work and made all the images around 70% smaller while keeping the right quality.

### CODE REVIEW

**Comment by bruvv** [comment]
Argh I see it also merged the main branch. Will close this: https://github.com/alexbelgium/hassio-addons/pull/136 and keep this as main pull request.

**Comment by alexbelgium** [comment]
Is this ready should I merge it? Thanks!

**Comment by bruvv** [comment]
Wait 1 sec, made a small error.

**Comment by bruvv** [comment]
@alexbelgium save to merge!

File: addons_updater/rootfs/scripts/00-aaa_dockerfile_backup.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/with-contenv bashio
```
Comment (alexbelgium) [inline addons_updater/rootfs/scripts/00-aaa_dockerfile_backup.sh]: Hi, you can't use bashio here - the point is to install it if the dockerfile build did not work. It is needed in context such as lock builds on supervised installations where curl doesn't seem to work. This was tested with several users of my addon and this was the only working solution to allow local builds on supervised installs. But then it requires #!/bin/bash

Comment (bruvv) [reply addons_updater/rootfs/scripts/00-aaa_dockerfile_backup.sh]: that is not good, sorry will change it all asap!

**Comment by bruvv** [comment]
fixed sorry @alexbelgium

**Comment by alexbelgium** [comment]
Hi, I don't know if you received a notification as they were above your last comment, but there were too last questions above. The main one is relative to the code structure to export a variable as ENV, I thought the initial code was valid (and at least it worked) and wanted to double check your proposal with you. Thanks!

**Comment by bruvv** [comment]
I cannot find those comments. :( as far as I am awair this can be merged now.

**Comment by alexbelgium** [comment]
The review comments appear above in the thread on my Web browser : I opened a review on the files modified and those comments are marked as pending 
- hassio-addons/zzz_templates/automatic_packages.sh
Sorry this needs to be bash too - this is the script called by 00-dockerfile_backup to install bashio (lines 157+)
- /00-global_var.sh
Are you sure that it is export line and not $line?
As the actual content of $line is something like VARIABLE=value, si I think we need to expand the variable line for the true variable VARIABLE to be exported. Do you see, what I mean?

again thanks very much for the additions and improvments!

**Comment by bruvv** [comment]
Fixed the bin bash stuff.
About the export line you can read more on this here:
https://github.com/koalaman/shellcheck/wiki/SC2163

**Comment by alexbelgium** [comment]
> Fixed the bin bash stuff. About the export line you can read more on this here: https://github.com/koalaman/shellcheck/wiki/SC2163

Hi, but this is my point : I do not want to export the variable line, but the variable that is in text inside.

line contains "VAR=value", so once $line is expanded, what the bash code sees is ```export VAR=value```

Example:
line="VAR=key"
export $line is therefore equivalent to export VAR=key

Do you see the rational?

**Comment by bruvv** [comment]
Ah I see! I'm not home ATM so I will do this later tonight.

**Comment by alexbelgium** [comment]
BTW you changed also the 00-global_var.sh script from bashio to bash but that one was correct :) as bashio is called within the script (bashio::config.false)

**Comment by alexbelgium** [comment]
Actually there is only two scirpts that can't accept the bashio shebang : 00_dockerfile-something, that is used to install elements when the dockerfile install failed for supervised installs, and the automatic_packages.sh that is the only that actually installs bashio :) it can be a bit confusing but that is the way I standardized the addons logic :)

**Comment by bruvv** [comment]
Ah yes pretty new to the whole basshio. So what you suggest to change for the merge ?

**Comment by alexbelgium** [comment]
> Ah yes pretty new to the whole basshio. So what you suggest to change for the merge ?

I think everything is fine except for export $line and to revert to binbash for global var :) I'm not at home but that's the only remaing things I was seeing :)

**Comment by bruvv** [comment]
Ah check I've already reverted the bash stuff in my last commits. Can you check them? I only have to change the line variable.

**Comment by alexbelgium** [comment]
Looks great!

**Comment by alexbelgium** [comment]
I've corrected the last few bashio and $line, let's merge!

Thanks again for all the proposals and improvements!

**Comment by bruvv** [comment]
Awesome thanks for the commits :)

**Pull request merged**